cilium-cli/connectivity/check/context.go

```diff
@@ -1297,5 +1297,7 @@ func (ct *ConnectivityTest) ForEachIPFamily(hasNetworkPolicies bool, do func(fea
 func (ct *ConnectivityTest) ShouldRunConnDisruptNSTraffic() bool {
 	return ct.params.IncludeConnDisruptTestNSTraffic &&
 		ct.Features[features.NodeWithoutCilium].Enabled &&
-		(ct.Params().MultiCluster == "" || ct.Features[features.KPRNodePort].Enabled)
+		(ct.Params().MultiCluster == "" || ct.Features[features.KPRNodePort].Enabled) &&
```

Comment:
Should we squash this into the first commit?

Reply:
I will revert this commit once I fixed the issues on XDP and ipsec. So let me keep it separate.